widgets/features/features.php

```diff
@@ -194,20 +194,20 @@ function get_widget_form(){
 
 			'container_shape' => array(
 				'type' => 'select',
-				'label' => __('Icon Container Position', 'so-widgets-bundle'),
+				'label' => __( 'Icon container position', 'so-widgets-bundle' ),
```

Comment:
This should be _shape_, shouldn't it? I.e. _Icon container **shape**_

Reply:
That would align with the setting name but I don't think we should change this however as Shape isn't as obvious as position. While it may make sense for us to change it, it will cause confusion. "What is a Top Shape?"

Reply:
Oh my mistake.